src/gui/windows/controls.zig

```diff
@@ -105,16 +106,26 @@ pub fn onOpen() void {
 	if(!editingKeyboard) {
 		list.add(ContinuousSlider.init(.{0, 0}, 256, 0, 5, main.settings.controllerAxisDeadzone, &updateDeadzone, &deadzoneFormatter));
 	}
-	for(&main.KeyBoard.keys) |*key| {
-		const label = Label.init(.{0, 0}, 128, key.name, .left);
-		const button = if(key == selectedKey) (Button.initText(.{16, 0}, 128, "...", .{})) else (Button.initText(.{16, 0}, 128, if(editingKeyboard) key.getName() else key.getGamepadName(), .{.callback = if(editingKeyboard) &keyFunction else &gamepadFunction, .arg = @intFromPtr(key)}));
-		const unbindBtn = Button.initText(.{16, 0}, 64, "Unbind", .{.callback = &unbindKey, .arg = @intFromPtr(key)});
-		const row = HorizontalList.init();
-		row.add(label);
-		row.add(button);
-		row.add(unbindBtn);
-		row.finish(.{0, 0}, .center);
-		list.add(row);
+	
+	for(std.enums.values(Window.Key.Group)) |group| {
+		const groupDisplayLabel = Label.init(.{0, 0}, 128, group.displayName(), .center);
```

Comment:
"Creative Controls" looks bad when split across multiple lines. Increase `maxWidth`?

```suggestion
		const groupDisplayLabel = Label.init(.{0, 0}, 320, group.displayName(), .center);
```

`128` (label width) + `128` (button width) + `64` (unbind button width) = `320` should be fine.